src/RecoveryServices/RecoveryServices.Backup.Test/RecoveryServices.Backup.Test.csproj

```diff
@@ -10,12 +10,12 @@
 		<RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)$(AzTestAssemblySuffix)</RootNamespace>
 	</PropertyGroup>
 
-	<ItemGroup>
-		<PackageReference Include="Microsoft.Azure.Management.Compute" Version="52.0.0" />
-		<PackageReference Include="Microsoft.Azure.Management.Network" Version="21.0.0" />
-		<PackageReference Include="Microsoft.Azure.Management.RecoveryServices" Version="4.3.2-preview" />
-		<PackageReference Include="Microsoft.Azure.Management.RecoveryServices.Backup" Version="4.4.0-preview" />
-	</ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="54.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="21.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.RecoveryServices" Version="4.3.2-preview" />
+	<PackageReference Include="Microsoft.Azure.Management.RecoveryServices.Backup" Version="4.3.0-preview" />
```

Comment:
This looks like it's accidently downgraded from 4.4.0-preview to 4.3.0-preview. Also the indentation is not consistent with the rest.